Drafts/scs-XXXX-v0-naming-conventions.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Recommended naming for domains/groups/roles/project when onboarding new customers
+type: _Standard | Decision Record_
+status: Draft
+track: IAM
+---
+
+<!---
+This is a template striving to provide a starting point for
+creating a standard or decision record adhering to scs-0001.
+Replace at least all text which is _italic_.
+See https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md
+--->
+
+## Introduction
+
+When CSPs try to enroll a new customer they encounter themselves in
+a situation where they have to choose names for the openstack domain,
+project and user.
+
+There might also exists some group or roles from the customers point of
+view, like `domain admin` or `project admin` not related at all with
+openstack.
+
+## Motivation
+
+Create a naming convention to use during the provisioning of the users and
+groups through an openstack domain.
+
+## Design Considerations
+
+OPTIONAL
+
+### Options considered
+
+#### PS approach to naming
+
+For naming the customers the suggestion from PS is the following:
+
+A prefix will be use to differenciate domain, project and user in
+the openstack environment. The project name is also added as a sufix.
+
+So the onboaring tool will create the following structure for a new
+customer onboarded in the system.
+
+```commandline
+domain: d<customer_id> 
+project: p<customer_id>-<project_name>
+user: u<customer_id>-<user_name>
+```
+
+For the customer also a domain admin group and a project admin group are
+created. These are not related in any way with OpenStack admin role. This groups use
```

Comment:
We can explain it maybe at this point:

```suggestion
created. Please note that, at the time of writing, PCO is not making any use of OpenStack domains to isolate customers. So, for the scope of this document, the term "domain admin" is used in a conceptual way rather than referring to the specific OpenStack implementation. For the scope of this document "domain admins" can do things like creating projects, creating users and granting users access to some projects (within the domain of course). "Users" can create and view resources within the projects that they have been granted access to by the "domain admin"

These groups use
```